website/meta/universe.json

```
@@ -4119,6 +4119,33 @@
             },
             "category": ["pipeline", "research"],
             "tags": ["Thai"]
+        },
+        {
+            "id": "vetiver",
+            "title": "Vetiver",
+            "slogan": "Version, share, deploy, and monitor models.",
+            "description": "The goal of vetiver is to provide fluent tooling to version, deploy, and monitor a trained model. Functions handle creating model objects, versioning models, predicting from a remote API endpoint, deploying Dockerfiles, and more.",
+            "github": "rstudio/vetiver-python",
+            "pip": "vetiver",
+            "code_example": [
+                "from vetiver import VetiverModel, VetiverAPI",
+                "",
```

```suggestion
                "import spacy",
                "from vetiver import VetiverModel, VetiverAPI",
                "",
                "# If you use this model, you'll need to download it first:",
                "# python -m spacy download en_core_web_sm",
                "nlp = spacy.load('en_core_web_md')",
```

I suggest adding a pipeline definition to make the example work out-of-the-box.